docs/reference/network-config.md

```diff
@@ -0,0 +1,22 @@
+---
+id: network-config
+---
+
+# Network configurations
+
+| Network     | RPC Endpoint                       | Chain ID      | 
+|-------------|------------------------------------|---------------|
+| Portal Loop | https://rpc.gno.land:443           | `portal-loop` |
+| Testnet 4   | upcoming                           | upcoming      |
+| Testnet 3   | https://rpc.test3.gno.land:443     | `test3`       |
+| Staging     | https://rpc.staging.gno.land:36657 | `test3`       |
+| Local       | http://127.0.0.1:26657             | `dev`         |
```

Comment:
Are you sure? I think it's still good to have this in the docs, since a local node does not display the listen port in an easy to see way (only in flags). Doesn't hurt to keep it here.

Reply:
Yes, feel free to drop

This info is something you get from your local node, not the docs

Reply:
[d3ed2fb](https://github.com/gnolang/gno/pull/1867/commits/d3ed2fb65a1df92201ad1dfc6392a5801162cad6)